.github/workflows/wheels-build.yaml

```diff
@@ -18,6 +18,8 @@ on:
       script:
         required: true
```

Comment:
Should this input be switched to not required?

Reply:
Yeah looks like that should be removed.